platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -425,6 +425,18 @@ public void onDestroy() {
     }
   }
 
+  /**
+   *  The preferred frame rate at which the map view is rendered,
+   *  but it can't excess the ability of device hardware.
+   *
+   * @param preferredFramesPerSecond  Can be set to arbitrary integer values.
+   */
+  public void setPreferredFramesPerSecond(int preferredFramesPerSecond) {
```

Comment:
Instead of exposing a setter directly on MapView. Can we follow the `MapboxMapOptions` paradigm instead? This concretely means integrating this configuration as an attribute and as a member in `MapboxMapOptions`.

Naming wise I feel `MapboxMapOptions#setMaximumFramesPerSecond` or `MapboxMapOptions#setMaximumFps` follows the other configurations more closely.

Reply:
Moved to MapboxOptions.